scripts/validate_systs.py

```diff
@@ -0,0 +1,382 @@
+  
+  Usage:
+  >> python OutlierPlot.py --signal_template <signal_template_{a}_{b}_{c}_for_masses> --x_var <which variable in signal name template to plot on x axis (defaults to 'a')> --y_var <which variable in signal name template to plot on x axis (defaults to 'b')> --v_max <max colourbar amplitude> --x_label <x axis label> --y_label <y axis label>
+  
+  Example for 1Lbb Wh analysis (https://glance.cern.ch/atlas/analysis/analyses/details.php?id=2969):
+  
+  >> python OutlierPlot.py --signal_template C1N2_Wh_hbb_{a}_{b} --x_var a --y_var b --v_max 10 --x_label '$m(\tilde{\chi}_{1}^{\pm}/\tilde{\chi}_{2}^{0})$ [GeV]' --y_label '$m(\tilde{\chi}_{1}^{0})$ [GeV]'
+  
+  The signal template is the name of an arbitrary signal in the json patch files, with the signal masses left as {}. Given a background-only file and a patch file, the signal name can be found under "samples" in the output of:
+  >> jsonpatch BkgOnly.json patch_XXX.json | pyhf inspect
+  
+  If, for example, one of the signals is called C1N2_Wh_hbb_550_200, where 550 and 200 are the variable model masses, the signal template would be C1N2_Wh_hbb_{}_{}.
+  """
+
+import json
+import glob
+import numpy as np
+import matplotlib.pyplot as plt
+import scipy.interpolate
+import parse
+import click
+import pyhf
+
+plt.rc("xtick", labelsize=14)
+plt.rc("ytick", labelsize=14)
+
+
+def handle_deltas(delta_up, delta_dn):
+    nom_is_center = np.bitwise_or(
+        np.bitwise_and(delta_up > 0, delta_dn > 0),
+        np.bitwise_and(delta_up <= 0, delta_dn <= 0),
+    )
+    span = delta_dn + delta_up
+    maxdel = np.maximum(np.abs(delta_dn), np.abs(delta_dn))
+    abs_unc = np.where(nom_is_center, span, maxdel)
+    return abs_unc
+
+
+def process_patch(p):
+    nom = np.asarray(p["value"]["data"])
+
+    # histosys
+    hid = np.asarray(
+        [
+            m["data"]["hi_data"]
+            for m in p["value"]["modifiers"]
+            if m["type"] == "histosys"
+        ]
+    )
+    lod = np.asarray(
+        [
+            m["data"]["lo_data"]
+            for m in p["value"]["modifiers"]
+            if m["type"] == "histosys"
+        ]
+    )
+    delta_up = hid - nom
+    delta_dn = nom - lod
+    histo_deltas = handle_deltas(delta_up, delta_dn)
+
+    hi = np.asarray(
+        [m["data"]["hi"] for m in p["value"]["modifiers"] if m["type"] == "normsys"]
+    )
+    lo = np.asarray(
+        [m["data"]["lo"] for m in p["value"]["modifiers"] if m["type"] == "normsys"]
+    )
+    delta_up = np.asarray([h * nom - nom for h in hi])
+    delta_dn = np.asarray([l * nom - nom for l in lo])
+    norm_deltas = handle_deltas(delta_up, delta_dn)
+
+    """
+    delta = np.asarray(
+        [m["data"] for m in p["value"]["modifiers"] if m["type"] == "staterror"]
+    )
+    delta_up = delta / 2.0
+    delta_dn = delta / 2.0
+    stat_deltas = handle_deltas(delta_up, delta_dn)
+    """
```

Comment:
Is there a reason to include commented code like this? Might be better to just remove it.
```suggestion
```